src/Routes.js

```diff
@@ -66,7 +66,8 @@ const Routes = (props) => (
     <Route path="/nih_dms_policy" component={NIHDMSPolicyInfo} />
     <Route path="/anvil_dms_policy" component={AnVILDMSPolicyInfo} />
     <AuthenticatedRoute path="/profile" component={UserProfile} props={props} rolesAllowed={[USER_ROLES.all]} />
-    <AuthenticatedRoute path="/request_role" component={RequestRole} props={props} rolesAllowed={[USER_ROLES.all]} />
+    <AuthenticatedRoute path="/request_role" component={RequestForm} props={Object.assign({isRequestRolePage: true}, props)} rolesAllowed={[USER_ROLES.all]} />
+    <AuthenticatedRoute path="/request_lc" component={RequestForm} props={Object.assign({isRequestLCPage: true}, props)} rolesAllowed={[USER_ROLES.all]} />
```

Comment:
When using Object.assign, the safe thing to do is to use an empty object to mutate and then merge in others as required. For example, if there is a `isRequestLCPage` in props, then it will be overwritten with the value in props instead of `true` here. 
```suggestion
    <AuthenticatedRoute path="/request_role" component={RequestForm} props={Object.assign({}, props, {isRequestRolePage: true})} rolesAllowed={[USER_ROLES.all]} />
    <AuthenticatedRoute path="/request_lc" component={RequestForm} props={Object.assign({}, props, {isRequestLCPage: true})} rolesAllowed={[USER_ROLES.all]} />
```

Reply:
That makes sense, thank you!